Server/Server.Core/Server.Core.Static/Settings.hpp

```diff
@@ -1,52 +1,31 @@
 #pragma once
 
-#include "ArgParser.hpp"
-
-#include <PostgreRepositoryManager.hpp>
+namespace Server::Builder
+{
+struct ParamType
+{
+    constexpr static char* ServerPort  = "--serverport";
+    constexpr static char* HostAddress = "--hostaddr";
+    constexpr static char* DBName      = "--dbname";
+    constexpr static char* DBPort      = "--port";
+    constexpr static char* DBUser      = "--user";
+    constexpr static char* DBPassword  = "--password";
+};
 
 class Settings
 {
 public:
     Settings() = default;
 
-    Settings(const Settings&)            = default;
-    Settings(Settings&&)                 = default;
-    Settings& operator=(const Settings&) = default;
-    Settings& operator=(Settings&&)      = default;
-
-    explicit Settings(const ArgParser& parser)
+    inline Settings& SetValue(std::pair<std::string, std::string> value)
     {
-        _serverPort     = parser.getPair("--serverport").second;
-        _hostAddress    = parser.getPair("--hostaddr").second;
-        _dbName         = parser.getPair("--dbname").second;
-        _dbPort         = parser.getPair("--port").second;
-        _dbUser         = parser.getPair("--user").second;
-        _dbPassword     = parser.getPair("--password").second;
+        _settings.emplace(value);
```

Comment:
There may be a problem with double writing.

Reply:
In the last commit I have the next edition:
_settings.insert_or_assign(value.first, value.second);

Reply:
@KliKonNik look at this like at potential copy-paste issue. I could forget change key so method `insert_or_assign` just erase my previous correct parameter.